bentoml/_internal/frameworks/fastai.py

```diff
@@ -0,0 +1,306 @@
+                If model uses **mixed_precision**, then the results models will also be converted to FP32. Learn more about `mixed precision <https://docs.fast.ai/callback.fp16.html>`.
+        pickle_module (``types.ModuleType``, `optional`, default to ``cloudpickle``):
+            The pickle module to use for loading the model. This is passthrough to ``torch.load``.
+
+    Returns:
+        :code:`fastai.learner.Learner`:
+            The :code:`fastai.learner.Learner` model instance loaded from the model store or BentoML :obj:`~bentoml.Model`.
+
+    Example:
+    .. code-block:: python
+
+        import bentoml
+
+        model = bentoml.fastai.load_model("fai_learner")
+        results = model.predict({"input": "some input"})
+    """  # noqa
+    if not isinstance(bento_model, bentoml.Model):
+        bento_model = get(bento_model)
+
+    if bento_model.info.module not in (MODULE_NAME, __name__):
+        raise BentoMLException(
+            f"Model {bento_model.tag} was saved with module {bento_model.info.module}, failed loading with {MODULE_NAME}."
+        )
+
+    return t.cast(
+        Learner,
+        load_learner(
+            bento_model.path_of(MODEL_FILENAME), cpu=cpu, pickle_module=pickle_module
+        ),
+    )
+
+
+def save_model(
+    name: str,
+    learner: Learner,
+    *,
+    pickle_module: types.ModuleType = cloudpickle,
+    signatures: ModelSignaturesType | None = None,
+    labels: dict[str, str] | None = None,
+    custom_objects: dict[str, t.Any] | None = None,
+    metadata: dict[str, t.Any] | None = None,
+) -> Tag:
+    """
+    Save a :code:`fastai.learner.Learner` model instance to the BentoML model store.
+
+    Args:
+        name (``str``):
+            The name to give to the model in the BentoML store. This must be a valid
+            :obj:`~bentoml.Tag` name.
+        model (:code:`fastai.learner.Learner`):
+            The :code:`fastai.learner.Learner` model to be saved.
+        signatures (``dict[str, ModelSignatureDict]``, `optional`):
+            Signatures of predict methods to be used. If not provided, the signatures default to
+            ``predict``. See :obj:`~bentoml.types.ModelSignature` for more details.
+        labels (``dict[str, str]``, `optional`):
+            A default set of management labels to be associated with the model. An example is
+            ``{"training-set": "data-1"}``.
+        custom_objects (``dict[str, Any]``, `optional`):
+            Custom objects to be saved with the model. An example is
+            ``{"my-normalizer": normalizer}``.
+
+            Custom objects are currently serialized with cloudpickle, but this implementation is
+            subject to change.
+        metadata (``dict[str, Any]``, optional):
+            Metadata to be associated with the model. An example is ``{"bias": 4}``.
+
+            Metadata is intended for display in a model management UI and therefore must be a
+            default Python type, such as ``str`` or ``int``.
```

Comment:
Apparently I didn't properly link built-in types either :P (another thing to go into the template...)

```suggestion
            default Python type, such as :obj:`str` or :obj:`int`.
```